hyperspy/io.py

```diff
@@ -478,6 +478,9 @@ def save(filename, signal, overwrite=None, **kwds):
         ensure_directory(filename)
         if overwrite is None:
             overwrite = overwrite_method(filename)
+        if overwrite is False:
```

Comment:
ah I understand what you mean, then: 
´´´
if overwrite is False and os.path.isfile(filename):
   pass
else:
   write file... (line 485 onwards) 

´´´